bgpd/bgp_vty.c

```diff
@@ -18934,6 +18934,9 @@ bool bgp_config_inprocess(void)
 	return event_is_scheduled(t_bgp_cfg);
 }
 
+/* Max wait time for config to load before post-config processing */
+#define BGP_PRE_CONFIG_MAX_WAIT_SECONDS 600
```

Comment:
bgp? :)

Reply:
Yes - you'll note that the processing was there before. This is the result of the discussion at last week's FRR meeting. If we ever miss a config_end callback, we want to log an error.

Reply:
Then we need a separate commit for this?

Reply:
The feedback I got was that we should add the messages to IS-IS and BGP as well since it is a bug if we don't receive the config_end. I don't mind add these to separate PRs if that is the consensus.

Reply:
Yes, please, then I can approve :)